client/model.go

```diff
@@ -39,20 +39,20 @@ type ProjectCreatePayload struct {
 }
 
 type ConfigurationVariableSchema struct {
-	Type string   `json:"string"`
+	Type string   `json:"type"`
 	Enum []string `json:"enum"`
 }
 
 type ConfigurationVariable struct {
-	ScopeId        string                      `json:"scopId"`
+	ScopeId        string                      `json:"scopeId"`
 	Value          string                      `json:"value"`
 	OrganizationId string                      `json:"organizationId"`
 	UserId         string                      `json:"userId"`
 	IsSensitive    bool                        `json:"isSensitive"`
-	Scope          string                      `json:"scope"`
+	Scope          Scope                       `json:"scope"`
 	Id             string                      `json:"id"`
 	Name           string                      `json:"name"`
-	Type           int64                       `json:"type"`
+	Type           ConfigurationVariableType   `json:"type"`
```

Comment:
Maybe add tests on the serialization of these?

Reply:
https://github.com/env0/terraform-provider-env0/pull/115/commits/fa1968507f41c838973354bca80c34da8d1d233f